pkg/rancher-desktop/pages/snapshots/create.vue

```diff
@@ -140,11 +140,11 @@ export default Vue.extend<Data, Methods, Computed, never>({
           :maxlength="35"
         />
       </div>
-      <div class="field notes-field">
-        <label>{{ t('snapshots.create.notes.label') }}</label>
+      <div class="field description-field">
```

Comment:
Not in this PR, but you should take out the `:maxLength='35'` and let the backend decide whether a proposed name is too long or not.

Also, when I enter an oversize name it's quietly truncated without any feedback. This might not be what a user expects.

Reply:
Truncating a name would be a bug. It is fine to display an abbreviated name, so are you sure it was actually truncated, or just displayed truncated?

And what happens if you create a long name from the CLI; how does the GUI deal with that?

Reply:
I'm playing with this now. I'm display a truncated version of the name, because right now an oversized name breaks the UI (like the 'Close' button is hidden under the error message)

Reply:
And I'm going to file a bug on the obscured `Close` button

Reply:
The name was absolutely truncated. Any characters that are entered after char 35 are lost.

Reply:
As for entering a long name from the CLI,

I ran

`rdctl snapshot create <80-character-name>`

and the UI lists

```
<first 39 characters>...
```

Weird that the ellipsis is not selectable.